observation_portal/requestgroups/serializers.py

```diff
@@ -160,6 +166,26 @@ def validate(self, data):
             raise serializers.ValidationError(_(
                 'Currently only square binnings are supported. Please submit with bin_x == bin_y'
             ))
+
+        extra_param_max_exp_time = 0
+        for field, value in data.get('extra_params', {}).items():
```

Comment:
This is for the requirement that exposure_time == the max of the individual exposure times